.github/ISSUE_TEMPLATE/A-submit-software-for-review.md

```diff
@@ -96,6 +96,11 @@ This package:
 
 </details>
 
+## Author's commitment
+
+- [ ] I have read the [author guide](https://devdevguide.netlify.app/authors-guide.html) and intend to fully participate in the process in particular respond to reviews, or will do my best to 
+find a successor if I e.g. change jobs.
```

Comment:
```suggestion
find a successor if necessary (for example because of a change in jobs).
```

(Sorry, I'm just deeply indoctrinated to avoid euro-centric latinate abbreviations :smile: )

Reply:
Hey, that's really good to know, thank you!